seatunnel-core/seatunnel-core-flink/src/main/java/org/apache/seatunnel/core/flink/utils/CommandLineUtils.java

```diff
@@ -71,19 +67,6 @@ public static List<String> buildFlinkCommand(FlinkCommandArgs flinkCommandArgs,
         if (flinkCommandArgs.isCheckConfig()) {
             command.add("--check");
         }
-        // set System properties
-        flinkCommandArgs.getVariables().stream()
-          .filter(Objects::nonNull)
-          .map(String::trim)
-          .forEach(variable -> command.add("-D" + variable));
-
-        if (jobType.equals(FlinkJobType.JAR)) {
```

Comment:
The function parameter `jobType` could be removed.

Reply:
Done.